understanding/20/use-of-color.html

```diff
@@ -38,13 +38,26 @@ <h2>Intent of Use of Color</h2>
          only in color.
       </p>
       
-      <p>Examples of information conveyed by color differences: “required fields are red",
-         “error is shown in red", and “Mary's sales are in red, Tom's are in blue". Examples
-         of indications of an action include: using color to indicate that a link will open
-         in a new window or that a database entry has been updated successfully. An example
-         of prompting a response would be: using highlighting on form fields to indicate that
-         a required field had been left blank.
-      </p>
+      <p>Examples of common failures include:</p>
+      <ul>
+        <li><strong>Information conveyed by color differences:</strong> required fields are only identified
+         by changing their border to a red color; sales figures for different employees are only
+         visually distinguished by using different foreground colors( e.g. Mary's sales are in red,
+         Tom's are in blue).</li>
+        <li><strong>Indicating an action:</strong> using color to indicate
+         that a link will open in a new window or that a database entry has been updated successfully.</li>
+        <li><strong>Prompting a response:</strong> using highlighting on form fields to indicate that
+         a required field had been left blank.</li>
+        <li><strong>Distinguishing a visual element:</strong> links are only distinguished from surrounding text
+         using a subtle color difference (with a contrast difference that is less than 3:1); for a series of toggle
+         buttons, the pressed/activated buttons are only visually indicated by a difference in color;
+         the interactive element that currently has focus is only distinguished by changing the color
+         of the element's existing background or border. However, particularly in the case of state/focus
+         indication, note that if the visual indicator was not previously present, this does not count as
+         a change in color. For instance, if a focused element has a new border, or a new solid background
+         shape, that was not there in its unfocused state, this does not count as a change of color alone,
+         since the indicator was not presentation to begin with.</li>
+      </ul>
```

Comment:
```suggestion
        <li><strong>Distinguishing a visual element:</strong>
            <ul>
                <li>links are only distinguished from surrounding text using a subtle color difference (with a contrast difference that is less than 3:1);</li>
                <li>for a series of toggle buttons, the pressed/activated buttons are only visually indicated by a difference in color;</li>
                <li>the interactive element that currently has focus is only distinguished by changing the color of the element's existing background or border.</li>
         </ul>
     </li>
     <p>However, particularly in the case of state/focus indication, note that if the visual indicator was not previously present, this does not count as a change in color. For instance, if a focused element has a new border, or a new solid background shape, that was not there in its unfocused state, this does not count as a change of color alone, since the indicator was not presentation to begin with.</p>
```

Consistent with the change above